website/content/en/docs/reference/vrl/examples.md

```diff
@@ -4,7 +4,23 @@ short: Examples
 weight: 3
 ---
 
-Here you'll find a comprehensive list of all VRL program examples. These examples demonstrate the breadth of the language and its observability-focused facilities.
+Here you'll find a comprehensive list of all VRL program examples. These
+examples demonstrate the breadth of the language and its observability-focused
+facilities.
+
+## Try Using The REPL
+
+You can run these examples using the **VRL REPL** (Read–eval–print loop).
+
+Assuming you have Vector installed, you can run `vector vrl` to start the REPL.
```

Comment:
We could also (until there's a supported web playground again) direct people to a more ephemeral docker solution:

```shell
docker run -it --rm timberio/vector:0.22.X-distroless-libc vrl
```

Reply:
Didn't realize it was auto-merged already - but providing the docker based instructions still may be a nice option as people are more likely to have docker/podman installed than Vector.